### PR TITLE
Release candidate 0.7.0

### PR DESCRIPTION
**Pull request by pameladelgado**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,5 +1,89 @@
...
+- **Template simplification**: We have decoupled the CLI version from the base image and
+  made it easier to override in your own environments. See the `project template
+  README <https://github.com/SwissDataScienceCenter/renku-project-template/blob/master/README.md>`_
+  for details.
+
+- **Improved editor**: All text editing components are using an enhanced editor
+  that allows seamless switching between WYSIWYG and markdown.
+
+
+- **``renku save``**: the Renku CLI now features a ``save`` command that
```
Comment (lorenzo-cavazzi) [inline CHANGELOG.rst:37]: Not sure if this gets rendered correctly with `**''`

Screenshot_20201029_164146

**Review by lorenzo-cavazzi — DISMISSED** [review]
🚀

**Review by ableuler — DISMISSED** [review]
I would say let's 🚢 - it

**Comment by rokroskar** [comment]
There is a small detail that is not mentioned - importing datasets from renku projects doesn't currently work for datasets in private repos - should this be mentioned somewhere?

**Comment by pameladelgado** [comment]
> should this be mentioned somewhere?

yes, we should probably mention it somewhere. If this behavior was introduced in this release then it should go in the release notes, otherwise I would put it in the readthedocs part. Is there a plan to support this in the future?

**Review by ableuler — APPROVED** [review]

**Pull request merged**